### PR TITLE
Local Warehouse model was updated

### PR DESCRIPTION
**Pull request by ShahanAliyev**

### CODE REVIEW

File: codes.txt
Comment (mmzeynalli) [inline codes.txt:1]: Never upload temp files to git

File: to-do.txt
Comment (mmzeynalli) [inline to-do.txt:1]: Check TODO commenting style and extensions in VSCode to nicely visualize them

**Review by mmzeynalli — CHANGES_REQUESTED** [review]

File: to-do.txt
Comment (ShahanAliyev) [reply to-do.txt:1]: I thought that mentioned them in gitignore

Comment (ShahanAliyev) [reply to-do.txt:1]: I thought that I had mentioned them in gitignore

**Review by mmzeynalli — APPROVED** [review]

**Pull request merged**